### DISCUSSION TITLE
Use vicon as motion capture system and log lps-twr data

### DISCUSSION THREAD

**Discussion by easy98** [USC-ACTLab/crazyswarm#403] [General]
Hi,

I use vicon as motion capture system and add lps-deck to log lps variables like ranging.distance. But when I set motion capture system as vicon then add lps-deck, CF will directly crash on the ground. Is there anyway to do so?

**Comment by whoenig** [comment]
I think this set-up causes the onboard EKF to be fed with inconsistent data, which might cause it to diverge. You could comment out https://github.com/bitcraze/crazyflie-firmware/blob/master/src/modules/src/estimator_kalman.c#L346-L355 in the firmware to avoid that.
